.github/workflows/model-evaluation.yml

```
@@ -0,0 +1,201 @@
+name: Full Model Evaluation
+
+on:
+  workflow_dispatch:
+    inputs:
+      provision-and-deploy:
+        description: 'Whether to provision the deployment'
+        required: true
+        default: true
+        type: boolean
+      deploy-only:
+        description: 'Whether to deploy the application'
+        required: true
+        default: false
+        type: boolean
+      purge:
+        description: 'Whether to purge the deployment after evaluation'
+        required: true
+        default: false
+        type: boolean
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+    deploy-azure:
+      if: ${{ inputs.provision-and-deploy == true || inputs.deploy-only == true }}
+      uses: ./.github/workflows/azure-dev.yml
```

When you're not executing a script I don't think there's a reason to start your path with `./`
```suggestion
      uses: .github/workflows/azure-dev.yml
```

---

sure